### DISCUSSION TITLE
OpenRewrite support

### DISCUSSION THREAD

**Discussion by miakiller** [renovatebot/renovate#22657] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.10.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Dear @rarkins 
Saw you mentioned openRewrite in your post here https://github.com/renovatebot/renovate/issues/8804#issuecomment-1298774332

Any news in regards to having native support for this?

Currently we are trying to get openRewrite maven plugin executing as postUpgradeTasks, main issue here is that renovate docker image does not include a maven installation.

Thank you and Kind Regards
Mia

### What you tried so far.

Using openRewrite outside of renovate.

**Comment by rarkins** [comment]
You can install Maven into the image at runtime using e.g. `install-tool maven 3.9.1`

**Comment by HonkingGoose** [comment]
@miakiller you (or your team) should update to the latest version of Renovate. Your current version is from October 2022.
